comfy/ldm/audio/dit.py

```diff
@@ -228,9 +228,9 @@ def __init__(
             linear_in = GLU(dim, inner_dim, activation, dtype=dtype, device=device, operations=operations)
         else:
             linear_in = nn.Sequential(
-                Rearrange('b n d -> b d n') if use_conv else nn.Identity(),
```

Comment:
`Rearrange` is not defined in `dit.py`, I believe it's a misspelling of `rearrange` from einops.